indexing-service/src/test/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorStateTest.java

```diff
@@ -2675,13 +2763,28 @@ private static Map<String, Object> getProperties()
   }
 
   private static SeekableStreamSupervisorTuningConfig getTuningConfig()
+  {
+    return createSupervisorTuningConfigWithWorkerThreads(1);
+  }
+
+  private static SeekableStreamSupervisorTuningConfig createSupervisorTuningConfig()
+  {
+    return createSupervisorTuningConfigWithWorkerThreads(null);
+  }
+
+  /**
+   * Creates a {@link SeekableStreamSupervisorTuningConfig} with the given number of worker threads.
+   *
+   * @param workerThreads the number of threads to use, or null to imitate default settings.
+   */
```

Comment:
Nit: Not really needed (especially in tests) unless it's adding info that is not obvious from the method name and signature.